drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -332,7 +332,7 @@ enum wifi_nrf_status wifi_nrf_fmac_dev_add_zep(struct wifi_nrf_drv_priv_zep *drv
 		goto out;
 	}
 
-	LOG_INF("Firmware (v%d.%d.%d.%d) booted successfully\n",
+	LOG_DBG("Firmware (v%d.%d.%d.%d) booted successfully\n",
```

Comment:
This was discussed internally and the consensus was this should be `INF`, see https://github.com/nrfconnect/sdk-nrf/pull/10729#issuecomment-1538290492

Reply:
I agree, even I was arguing the same in that PR, just linking it as a reference.

Reply:
@sachinthegreen  FYI.

Reply:
@bama-nordic @udaynordic : FYI..